pkg/kv/kvserver/scheduler.go

```diff
@@ -242,6 +247,11 @@ func newRaftScheduler(
 		processor:      processor,
 		metrics:        metrics,
 	}
+
+	// Priority shard at index 0.
+	s.shards = append(s.shards, newRaftSchedulerShard(priorityWorkers, maxTicks))
+
+	// Regular shards.
 	numShards := 1
 	if shardSize > 0 && numWorkers > shardSize {
 		numShards = (numWorkers-1)/shardSize + 1
```

Comment:
Does `numShards` include or not include the priority shard?

With this arithmetics, it looks like `numShards` is at least 2 (`numWorkers > shardSize` => `numWorkers-1 >= shardSize` => `(numWorkers-1)/shardSize + 1 >= 2`).

So, with the appends loop below, we'll end up with at least 3 shards. But it looked like we want the min number of shards to be 2 (priority + non-priority)?